lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -2128,11 +2150,15 @@ impl WorkspaceSnapshotGraph {
                 } => {
                     let updated_source = self.get_latest_node_idx(source.index)?;
                     let destination = self.get_latest_node_idx(destination.index)?;
-                    self.remove_edge(
+                    self.remove_edge_inner(
                         to_rebase_change_set,
                         updated_source,
                         destination,
                         *edge_kind,
+                        // Updating the vector clocks here may cause us to pick
+                        // the to_rebase node incorrectly in ReplaceSubgraph, if
+                        // ReplaceSubgraph comes after this RemoveEdge
+                        false,
```

Comment:
changing this to true will cause the new test to fail

Reply:
It should be safe to not update the ordering node vector clock here, since if we get a RemoveEdge update for an ordered container, the ordering node's node hash will have changed, and thus we should always get a ReplaceSubgraph update for that ordering node (unless we produce conflicts instead)